src/pages/dashboard/nodeSignIn/__tests__/NodeSignInECForm.test.tsx

```diff
@@ -103,8 +109,15 @@ describe('NodeSignInECForm', (injectedHistory?: ReturnType<typeof createMemoryHi
     });
   });
 
-  test('Test rendering NodeSignInECForm with intermediary false and Sumbit', async () => {
-    renderApp();
+  test('Test rendering NodeSignInECForm with intermediary false and Sumbit an inderiect ec', async () => {
```

Comment:
testcase missing: in update form fields should be precompilated with some expected data before setupForm